docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.less

```diff
@@ -0,0 +1,25 @@
+.icon-search-container {
+    display: flex;
+
+    .icon-resize-container {
+        display: inline-flex;
+        align-items: center;
+
+        .icon-resize-text {
+            justify-content: center;
+            flex-direction: column;
+        }
+
+        .icon-resize-buttons {
+            margin-left: 8px;
+
+            .icon-resize-button {
+                text-transform: none;
+
+                &:first-child {
+                    margin-left: -1px;
```

Comment:
Is this required? Having it doesn't seem to add much benefit? Or is there a purpose for this?